src/main/java/Duke.java

```diff
@@ -1,10 +1,55 @@
+import java.util.Scanner;
+import java.time.format.DateTimeParseException;
+
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private Scanner in = new Scanner(System.in);
+    private CompilationOfFiles storage;
+    private ListOfTasks xs;
+    private Ui ui;
+
+    public Duke (String filepath) {
+        ui = new Ui();
+        storage = new CompilationOfFiles(filepath);
+        xs = new ListOfTasks();
```

Comment:
Maybe xs might not be explicit enough to represent tasklist, don't you think so?